### PR TITLE
feat: 🎸 Add google login

### PR DESCRIPTION
**Pull request by kamack38**

### CODE REVIEW

File: packages/frontend/src/pages/AuthPage/AuthPage.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const { token } = useParams()
+
+  useEffect(() => {
+    if (token) {
+      http.setToken(token)
+    }
+    navigate('../dashboards', { replace: true })
+  }, [token, navigate])
+
+  // const loginTroughOAuth = () => {
```
Comment (kacper-cyra) [inline packages/frontend/src/pages/AuthPage/AuthPage.tsx:18]: remove

**Review by kacper-cyra — CHANGES_REQUESTED** [review]
Nice 😄

File: packages/backend/src/infrastructure/express/router/task.routes.ts [hunk 2]
```diff
@@ -19,12 +20,12 @@ export class TaskRoutes extends CommonRoutesConfig {
 
     this.app
       .route(`${V1}/tasks`)
-      .get((req, res) => taskController.findAll(req, res))
-      .post((req, res) => taskController.createTask(req, res))
+      .get(passport.authenticate('jwt', { session: false }), (req, res) => taskController.findAll(req, res))
```
[the remark this answers is not in the document]
Comment (kamack38) [reply packages/backend/src/infrastructure/express/router/task.routes.ts:23]: How shall I move it?

[the remark this answers is not in the document]
Comment (kacper-cyra) [reply packages/backend/src/infrastructure/express/router/task.routes.ts:23]: Create function - `JwtGuard = () => passport.authenticate('jwt', { session: false })`
and use it as:
`.get(JwtGuard(), (req, res) => ...`

[the remark this answers is not in the document]
Comment (kamack38) [reply packages/backend/src/infrastructure/express/router/task.routes.ts:23]: Ok

**Review by kacper-cyra — APPROVED** [review]

**Pull request merged**